latest/ug/automode/auto-cis.adoc

```diff
@@ -0,0 +1,61 @@
+include::../attributes.txt[]
+
+[.topic]
+[#auto-cis]
+= Generate CIS compliance reports from Kubernetes nodes using kubectl debug
+:info_titleabbrev: Generate CIS report
+
+This topic describes how to generate CIS (Center for Internet Security) compliance reports for Amazon EKS nodes using the `kubectl debug` command.
+
+The command allows you to temporarily create a debugging container on a Kubernetes node and run CIS compliance checks using the `apiclient` tool. The `apiclient` tool is part of Bottlerocket OS, the OS used by EKS Auto Mode nodes. 
+
+== Prerequisites
+
+Before you begin, ensure you have:
+
+* Access to an Amazon EKS cluster with kubectl configured
+* The appropriate IAM permissions to debug nodes
+* A valid profile that allows debug operations (e.g., `sysadmin`)
```

Comment:
explain what type of profile

Reply:
These are the profiles, https://kubernetes.io/docs/tasks/debug/debug-application/debug-running-pod/#debugging-profiles . sysadmin is the only one that works due to the need to enter the namespace of init.